detectorSegmentations/src/GridDRcalo_k4geo.cpp

```diff
@@ -68,13 +70,64 @@ namespace DDSegmentation {
     auto rot = paramBase->GetRotationZYX(noPhi);
     auto translation = rot * localPos;
 
+    // we want the cell position to be the front end of the tower
+    // retrieve the fiber length
+    int row = y(cID);
+    int col = x(cID);
+    double fiberLen = paramBase->GetShortFibers(noEta).retrieveFiberLength(row, col);
```

Comment:
I found a bit confusing to always call this through the short fibers, while some fibers here have the full length. Can you add here a comment stating that this method returns the full length fiber dimension if the row col do not correspond to any short fiber?